src/main/webapp/app/shared/model/achievable-skill.model.ts

```diff
@@ -7,6 +7,7 @@ export interface IAchievableSkill {
     description?: string;
     achievedAt?: Moment;
     irrelevant?: boolean;
+    skillStatus?: string;
```

Comment:
My typescript heart is crying. We have nice enum types in TS:
https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
Frontend now uses enum.